docs/developer-docs/developer-welcome.md

```diff
@@ -0,0 +1,181 @@
+4. Check the achievements you’d like to promote (Be careful to exclude any unwanted or voided achievements if there are any, otherwise just hit “All”)
+5. Click Bulk promote or demote
+6. Click Promote selected
+
+![Achievements being promoted through the Management site](/public/management-achievement-promotion.png)
+
+## Achievement-News Announcement
+
+Now that your set is live, it is time to announce it to everyone if you are on Discord!
+
+We have a handy bot command that you can use to auto generate a template *as long as your set is in core*. Go to #bot-spam and type in “!gan [gameid]” where [gameid] is the id of your game. 
+
+The pinned messages in #devs-chat have an alternate colored template that can be used as well.
+
+You can copy the results, replace the @{AUTHOR_NAME} with your name, and post it in the #achievement-news channel with an @Achievement-News ping to get everyone's attention on the hottest set that just dropped on RA.
+
+## Claims
+
+Now that you are a developer, the claim limit has expanded to a total of 4 concurrent regular claims. As long as you have no active tickets, you are free to claim sets. For more specific details look over the [claims section of the Developer Guidelines](/guidelines/developers/claims-system).
+
+If the game does not yet have a page, see [Creating Game Pages and Linking Hashes](#creating-game-pages-and-linking-hashes) for guidance.
+
+## Set Revisions
+
+Achievement set revisions are an important part in keeping older sets up-to-date with modern standards. This is your chance to ensure that quality is maintained on the site. Votes are done through the #revision-voting and #revision-discussion channels of the Discord server. You will be given the @Revision-Voting and @Rescore roles by default upon graduation.
+
+When considering whether to vote to approve a revision proposal, please familiarize yourself with the revision goal criteria outlined in the discussion forum pinned post. The goal of a revision is to turn a lacking set into a high-quality, modern set that ideally will never need another revision. Voting in revision proposals is an important responsibility that all developers should take seriously and participate in for the long-term health of the RetroAchievements project.
+
+When proposing a revision, it requires contacting all active authors of the relevant set. If you wish to Opt-Out of being contacted, use [this form](https://forms.gle/mgzv7RHbJEPCrxc77).
+
+For details on how to propose a revision, see [Achievement Set Revisions](/guidelines/content/achievement-set-revisions)
+
+If you are re-authoring the set (keeping the old achievements in Unofficial and promoting your new reworked achievements), make sure to contact DevCompliance for approval. If you want to transfer the earners for some or all of the previous achievements, you will need to reach out to RAdmin or #admin-site-cleanup on Discord for assistance.

```

Comment:
I would reword this to "If you are considering requesting a re-author of the set... ...contact DevCompliance for approval." and include a note to not put reauthoring in the revision proposal.